mkdocs.yml

```diff
@@ -29,69 +29,64 @@ markdown_extensions:
-        - Lucene: OrchardCore.Modules/OrchardCore.Lucene/README.md
-        - Queries: OrchardCore.Modules/OrchardCore.Queries/README.md
-        - Taxonomies: OrchardCore.Modules/OrchardCore.Taxonomies/README.md
-        - Templates: OrchardCore.Modules/OrchardCore.Templates/README.md
-        - Themes: OrchardCore.Modules/OrchardCore.Themes/README.md
-        - Liquid: OrchardCore.Modules/OrchardCore.Liquid/README.md
-        - Custom Settings: OrchardCore.Modules/OrchardCore.CustomSettings/README.md
-        - Widgets: OrchardCore.Modules/OrchardCore.Widgets/README.md
-        - Flows: OrchardCore.Modules/OrchardCore.Flows/README.md
-        - Forms: OrchardCore.Modules/OrchardCore.Forms/README.md
-        - Admin Menu: OrchardCore.Modules/OrchardCore.AdminMenu/README.md
-    - Core Modules:
-        - Configuration: OrchardCore/OrchardCore/Environment/Shell/README.md
-        - Data: OrchardCore/OrchardCore.Data/README.md
-        - Dynamic Cache: OrchardCore.Modules/OrchardCore.DynamicCache/README.md
-        - GraphQL: OrchardCore.Modules/OrchardCore.Apis.GraphQL/README.md
-        - GraphQL queries: OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/README.md
-        - Health Check: OrchardCore.Modules/OrchardCore.HealthChecks/README.md
-        - Localization: OrchardCore.Modules/OrchardCore.Localization/README.md
-        - Logging Serilog: OrchardCore/OrchardCore.Logging.Serilog/readme.md
-        - OpenId: OrchardCore.Modules/OrchardCore.OpenId/README.md
-        - Placement: OrchardCore/OrchardCore.DisplayManagement/README.md
-        - Recipes: OrchardCore.Modules/OrchardCore.Recipes/README.md
-        - Resources: OrchardCore.Modules/OrchardCore.Resources/README.md
-        - Roles: OrchardCore.Modules/OrchardCore.Roles/README.md
-        - Scripting: OrchardCore.Modules/OrchardCore.Scripting/README.md
-        - Setup: OrchardCore.Modules/OrchardCore.Setup/README.md
-        - Tenants: OrchardCore.Modules/OrchardCore.Tenants/README.md
-        - Workflows: OrchardCore.Modules/OrchardCore.Workflows/README.md
-        - DataProtection (Azure Storage): OrchardCore.Modules/OrchardCore.DataProtection.Azure/README.md
-    - Authentication Modules:
-        - Microsoft: OrchardCore.Modules/OrchardCore.Microsoft.Authentication/README.md
-        - Facebook: OrchardCore.Modules/OrchardCore.Facebook/README.md
-        - Twitter: OrchardCore.Modules/OrchardCore.Twitter/README.md
-        - GitHub: OrchardCore.Modules/OrchardCore.GitHub/README.md
-        - Google: OrchardCore.Modules/OrchardCore.Google/README.md
+        - Create a modular application: docs/guides/create-modular-application-mvc/index.md
+        - Run code on startup: docs/guides/run-code-on-startup/index.md
+        - Create a website: docs/guides/create-cms-application/index.md
+        - Add an item to the admin menu: docs/guides/add-admin-menu/index.md
+        - Install localization files: docs/guides/install-localization-files/index.md
+    - Modules:
```

Comment:
Does it make sense to have "Modules" and "Core" when they are all modules? Just that some are for CMS and the others not.

Reply:
A tag on the menu name could work here.